Sources/Rownd/framework/Customizations.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  Customizations.swift
+//  Rownd
+//
+//  Created by Matt Hamann on 9/12/22.
+//
+
+import Foundation
+import SwiftUI
+
+public struct RowndCustomizations: Encodable {
+    public init(){}
+
+    public var sheetBackgroundColor: UIColor = UIColor(named: "SheetBackgroundColor", in: getFrameworkBundle(), compatibleWith: .current)!
```

Comment:
!

Reply:
I had to reload the Rownd framework. It's working for me now

Reply:
Yeah not sure why this crashes the app for rme. Same error that you have, but I've reloaded the framework to no avail.

Reply:
Okay that's not good. I'm importing the framework differently so that's probably why it's working for me

Reply:
?

Reply:
@BobbyRadford how are you importing the framework? via SPM?

Reply:
yeah